code-experiments/build/c/meson.build

```diff
@@ -0,0 +1,20 @@
+project('example_experiment',
+  default_options: ['warning_level=3', 'buildtype=release']
+  )
+
+add_languages('c')
```

Comment:
The idiomatic way to add a language is in the project function.

```meson
project('example_experiment', 'c', default_options: ['warning_level=3'])
```

(Hmm, why isn't the cmake build also setting `-Wall -Wextra -Wpedantic`?)

Reply:
Thanks for the feedback! Changed the language specification to the more idiomatic way. Also tried to add warnings to the CMakeLists.txt, but afaik there's no easy and compiler agnostic way to do this.